versioned_docs/version-4.4/general-concepts/acl-permissions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: ACL Permissions
+---
+ACL stands for Access Control List.
+
+ACL Access relates to what users can view.
+
+ACL Permissions relate to what users can do. 
+
+Joomla has a very sophisticated permissions framework, and if you're not familiar with it you should view some introductory material first. I'd recommend the [Access Control List Tutorial](https://docs.joomla.org/J3.x:Access_Control_List_Tutorial) and the video [Joomla 3 ACL Explained with Randy Carey](https://www.youtube.com/watch?v=CFqXAc3orkY) (between 2 minutes and 32 minutes is the section to watch). Both of these related to Joomla 3, but ACL hasn't changed between Joomla version 3 and version 4 or 5.
+
+# Permissions - an implementation view
+Permissions are held in the Joomla `#__assets` table, in the `rules` column. If you have a clean Joomla install, in the `assets` record for `com_content` (ie the record where the `name` column is "com_content") you'll have an entry in the `rules` column similar to (but probably not exactly the same as):
+
+{"core.admin":{"7":1},"core.manage":{"6":1},"core.create":{"3":1},"core.edit":{"4":1,"2":1},"core.edit.state":{"5":1},"core.execute.transition":{"6":1,"5":1},"core.delete":{"2":0}}
+

```

Comment:
Docusaurus 3.0 uses a new version of the markdown parser (which also can execute javascript and do other things) so it's a bit picky about `{` characters because they are used for variables.

Can you encapsulate this string with \`

Beside that we also should encapsulate such strings with code highlighting.

for this I would use
\```json
{"core.admin":{"7":1},"core.manage":{"6":1},"core.create":{"3":1},"core.edit":{"4":1,"2":1},"core.edit.state":{"5":1},"core.execute.transition":{"6":1,"5":1},"core.delete":{"2":0}}
\```

and maybe make it more readable with line breaks